spec/aruba/aruba_path_spec.rb

```diff
@@ -98,6 +98,6 @@
     before(:each) { FileUtils.mkdir_p File.dirname(new_path) }
     before(:each) { File.open(new_path, 'w') { |f| f.print 'a' } }
 
-    it { expect(path.blocks).to be > 0 }
+    it { expect(path.blocks || 123).to be > 0 }
```

Comment:
As far as I know `File.blocks` and the like are not supported on windows. We never made it to the point where we made the specs aware of different operating systems/filesystems - we also started a discussion about that on [gitter](https://gitter.im/cucumber/aruba?at=56f8464de4a8384a1bbc6b3e) some days ago. What I think we need to do is, to fix the specs properly by only testing things on platforms where they are supposed to work.